### PR TITLE
Add OSTN02 link to OS website

### PR DESCRIPTION
**Pull request by snodnipper**
inc. brief explanation.

### CODE REVIEW

**Comment by snodnipper** [comment]
@dhardiman we should get slack notifications...didn't just now :( - and I did press that gorgeous green button.

File: README.md [hunk 1]
```diff
@@ -10,3 +10,9 @@ Includes examples using:
 * [`ogr2ogr`](http://www.gdal.org/ogr2ogr.html) - single command + small config
 * `C++` - user input is transformed and output with printf
 
+# OSTN02 #
+The British National Grid (BNG) is not a perfect grid because it was made prior to modern high accuracy technology.  It can be likened to a grid made from string, where certain parts are slightly pulled to signify inaccuracy.
+
+In its original form OSTN02 consists of a plane 700km by 1250km grid of 1km resolution with an eastings shift and northing shift at each grid node.  The OSTNS02 file corrects the funamental BNG inaccuracies, which enables coordinate values to be transformed to other projections (e.g. EPSG:4326) via mathematical function whilst maintaining accuracy to the true location.
```
Comment (jskeates) [inline README.md]: A couple of SPAG issues:`funamental` and `via mathematical function` (could be `via a mathematical function` or `via mathematical functions`)

**Review by jskeates — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (snodnipper) [inline README.md]: +1 - roger that.  Updated.

Comment (snodnipper) [inline README.md]: With luck we can change this to OSTN15 soon.

**Review by jskeates — APPROVED** [review]

**Pull request merged**